packages/wallets/src/components/AppCard/AppCard.tsx

```diff
@@ -8,14 +8,14 @@ import './AppCard.scss';
 
 type TProps = {
     activeWalletCurrency?: THooks.ActiveWalletAccount['currency'];
-    appName?: string;
+    appName?: JSX.Element | string;
```

Comment:
Just a minor nitpick from the initial PR that @lubega-deriv he would type it as `ReactNode` instead of this because it also accepts strings. Maybe we should keep it consistent and use the same type across the project.

Reply:
mmm the type of ReactNode as for me is like any: `type ReactNode = ReactElement | string | number | boolean | null | undefined | ReactFragment | ReactPortal;`
but we want to render string(localize) or JSX.Element(Localize) so IMHO we should narrow our types to avoid passing boolean, null, undefined, etc 🤔

Reply:
you got a point there 🤔